system-tests/lib/resultsUtils.ts

```diff
@@ -240,25 +240,23 @@ export const expectCorrectModuleApiResult = (json, opts: {
     expectStartToBeBeforeEnd(run, 'stats.startedAt', 'stats.endedAt')
     expectStartToBeBeforeEnd(run, 'reporterStats.start', 'reporterStats.end')
 
-    const attempts = _.flatMap(run.tests, (test) => test.attempts)
-
-    const wallClocks = _.sumBy(attempts, 'duration')
+    const wallClocks = _.sumBy(run.tests, 'duration')
 
     // ensure each run's duration is around the sum
     // of all tests wallclock duration
     expectDurationWithin(
       run,
       'stats.duration',
       wallClocks,
-      wallClocks + 400, // add 400ms to account for padding
+      wallClocks + 600, // add 600ms to account for padding
       1234,
     )
 
     expectDurationWithin(
       run,
       'reporterStats.duration',
       wallClocks,
-      wallClocks + 400, // add 400ms to account for padding
+      wallClocks + 600, // add 600ms to account for padding
```

Comment:
any idea why we needed to increase the padding here?

Reply:
I found it liable to be a bit flaky. I even saw this fail in an unrelated branch.